frontend/appflowy_flutter/lib/plugins/ai_chat/application/chat_bloc.dart

```diff
@@ -277,8 +282,16 @@ class ChatBloc extends Bloc<ChatEvent, ChatState> {
   }
 
   // Related questions handler
-  void _handleRelatedQuestions(List<String> questions) {
+  void _handleRelatedQuestions(
+    List<String> questions,
+    Emitter<ChatState> emit,
+  ) {
     if (questions.isEmpty) {
+      emit(
+        state.copyWith(
+          promptResponseState: PromptResponseState.ready,
+        ),
+      );
```

Comment:
I think this is no longer necessary because a different `PromptResponseState` is no longer being emitted.